app/src/main/java/com/example/githubusers/model/GithubUsersResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.example.githubusers.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+
+public class GithubUsersResponse {
+
+    @SerializedName("items")
+    private List<GithubUsers> items;
```

Comment:
Make this field describe what it contains.

Reply:
Okay